docker/Dockerfile

```diff
@@ -12,9 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.18-bullseye AS build
-
-RUN apt-get update && apt-get install -y --no-install-recommends apt-transport-https sudo
+FROM ubuntu:jammy-20220531 AS build
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gcc \
+    git \
+    golang \
```

Comment:
What is the golang version bundled with `ubuntu:jammy-20220531`? We need to make sure it's at least 1.18.

Reply:
1.18.1